testproject/Packages/manifest.json

```diff
@@ -1,15 +1,16 @@
 {
   "dependencies": {
     "com.unity.collab-proxy": "1.5.7",
-    "com.unity.ide.rider": "3.0.5",
+    "com.unity.ide.rider": "3.0.6",
     "com.unity.ide.visualstudio": "2.0.8",
     "com.unity.ide.vscode": "1.2.3",
     "com.unity.multiplayer.mlapi": "file:../../com.unity.multiplayer.mlapi",
     "com.unity.multiplayer.transport.utp": "file:../../com.unity.multiplayer.transport.utp",
     "com.unity.package-validation-suite": "0.19.2-preview",
-    "com.unity.test-framework": "1.1.24",
+    "com.unity.test-framework": "1.1.27",
+    "com.unity.test-framework.performance": "2.3.1-preview",
```

Comment:
are version number changes in this file intentional?

Reply:
good point, this happened while switching between 2020 and 2021. Will revert useless ones (but keep the test ones)